tests/golos.publication_rewards_tests.cpp

```diff
@@ -713,8 +738,8 @@ BOOST_FIXTURE_TEST_CASE(limits_test, reward_calcs_tester) try {
     );
     check();
     auto ref_block_num_bob3 = control->head_block_header().block_num();
-    BOOST_CHECK_EQUAL(err.limit_no_vesting, create_message({N(bob), "limit-no-vesting", ref_block_num_bob3}));
-    BOOST_CHECK_EQUAL(success(), create_message({N(bob1), "test2", ref_block_num_bob3}));
+    BOOST_CHECK_EQUAL(err.limit_no_vesting, create_msg({N(bob), "limit-no-vesting", ref_block_num_bob3}, 0));
+    BOOST_CHECK_EQUAL(success(), create_msg({N(bob1), "test2", ref_block_num_bob3}, 0));
```

Comment:
each `create_msg` call have `0` as 2nd argument. maybe make it default value if it unused?